src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -2540,11 +2558,79 @@ public static void findWelcomeFrame(RemoteRobot remoteRobot) {
         } catch (WaitForConditionTimeoutException e) {
             // If the welcome frame is not found then there is a project loaded.
             // Close the editor files and close the project to get back to the welcome frame.
-            UIBotTestUtils.closeAllEditorTabs(remoteRobot);
+            UIBotTestUtils.runActionFromSearchEverywherePanel(remoteRobot, "Close All Tabs", 3);
             UIBotTestUtils.closeProjectView(remoteRobot);
             UIBotTestUtils.closeProjectFrame(remoteRobot);
             TestUtils.sleepAndIgnoreException(30); // takes about 15s to render the whole Welcome page including the Open button on Mac.
             UIBotTestUtils.validateProjectFrameClosed(remoteRobot);
         }
     }
+
+    /**
+     * Clicks on the main menu button.
+     *
+     * @param remoteRobot The RemoteRobot instance.
+     */
+    public static void clickOnMainMenu(RemoteRobot remoteRobot) {
+        String fileName = "Main Menu";
```

Comment:
This doesn't look like a file name. It seems there could be a better name for this variable.

Reply:
I renamed the variable name. Thanks  @mrglavas